comp/trace/config/setup.go

```diff
@@ -341,6 +341,20 @@ func applyDatadogConfig(c *config.AgentConfig, core corecompcfg.Component) error
 		ProbabilisticSampling:  core.GetFloat64("otlp_config.traces.probabilistic_sampler.sampling_percentage"),
 	}
 
+	if core.IsSet("apm_config.install_id") {
+		c.InstallSignature.Found = true
+		c.InstallSignature.InstallID = core.GetString("apm_config.install_id")
+	}
+	if core.IsSet("apm_config.install_time") {
+		c.InstallSignature.Found = true
+		c.InstallSignature.InstallTime = core.GetInt64("apm_config.install_time")
+	}
+	if core.IsSet("apm_config.install_type") {
+		c.InstallSignature.Found = true
+		c.InstallSignature.InstallType = core.GetString("apm_config.install_type")
+	}
+	applyOrCreateInstallSignature(c)
```

Comment:
This can end up with us having a partial signature, I believe. e.g. `InstallType` but no time or id.

Reply:
True, I don't think it's that's a big issue because we'll still make some sense of the telemetry data with only one or two of them.
Also the goal is that these configs won't be set by users but either injected into the env/config by the operator/helm chat or set by the installer